src/libsystemd/libsystemd.sym

```diff
@@ -782,4 +782,173 @@ global:
+        sd_rtnl_message_link_get_ifindex;
+        sd_rtnl_message_link_get_flags;
+        sd_rtnl_message_link_get_type;
+        sd_rtnl_message_new_route;
+        sd_rtnl_message_route_set_dst_prefixlen;
+        sd_rtnl_message_route_set_src_prefixlen;
+        sd_rtnl_message_route_set_scope;
+        sd_rtnl_message_route_set_flags;
+        sd_rtnl_message_route_set_table;
+        sd_rtnl_message_route_set_type;
+        sd_rtnl_message_route_get_flags;
+        sd_rtnl_message_route_get_family;
+        sd_rtnl_message_route_get_protocol;
+        sd_rtnl_message_route_get_scope;
+        sd_rtnl_message_route_get_tos;
+        sd_rtnl_message_route_get_table;
+        sd_rtnl_message_route_get_dst_prefixlen;
+        sd_rtnl_message_route_get_src_prefixlen;
+        sd_rtnl_message_route_get_type;
+        sd_rtnl_message_new_nexthop;
+        sd_rtnl_message_nexthop_set_flags;
+        sd_rtnl_message_nexthop_get_flags;
+        sd_rtnl_message_nexthop_get_family;
+        sd_rtnl_message_nexthop_get_protocol;
+        sd_rtnl_message_new_neigh;
+        sd_rtnl_message_neigh_set_flags;
+        sd_rtnl_message_neigh_set_state;
+        sd_rtnl_message_neigh_get_family;
+        sd_rtnl_message_neigh_get_ifindex;
+        sd_rtnl_message_neigh_get_state;
+        sd_rtnl_message_neigh_get_flags;
+        sd_rtnl_message_new_addrlabel;
+        sd_rtnl_message_addrlabel_set_prefixlen;
+        sd_rtnl_message_addrlabel_get_prefixlen;
+        sd_rtnl_message_new_routing_policy_rule;
+        sd_rtnl_message_routing_policy_rule_set_tos;
+        sd_rtnl_message_routing_policy_rule_get_tos;
+        sd_rtnl_message_routing_policy_rule_set_table;
+        sd_rtnl_message_routing_policy_rule_get_table;
+        sd_rtnl_message_routing_policy_rule_set_fib_src_prefixlen;
+        sd_rtnl_message_routing_policy_rule_get_fib_src_prefixlen;
+        sd_rtnl_message_routing_policy_rule_set_fib_dst_prefixlen;
+        sd_rtnl_message_routing_policy_rule_get_fib_dst_prefixlen;
+        sd_rtnl_message_routing_policy_rule_set_fib_type;
+        sd_rtnl_message_routing_policy_rule_get_fib_type;
+        sd_rtnl_message_routing_policy_rule_set_flags;
+        sd_rtnl_message_routing_policy_rule_get_flags;
+        sd_rtnl_message_new_traffic_control;
+        sd_rtnl_message_traffic_control_get_ifindex;
+        sd_rtnl_message_traffic_control_get_handle;
+        sd_rtnl_message_traffic_control_get_parent;
+        sd_rtnl_message_new_mdb;
+
+        sd_nfnl_socket_open;
+        sd_nfnl_message_batch_begin;
+        sd_nfnl_message_batch_end;
+        sd_nfnl_nft_message_del_table;
+        sd_nfnl_nft_message_new_table;
+        sd_nfnl_nft_message_new_basechain;
+        sd_nfnl_nft_message_new_rule;
+        sd_nfnl_nft_message_new_set;
+        sd_nfnl_nft_message_new_setelems_begin;
+        sd_nfnl_nft_message_del_setelems_begin;
+        sd_nfnl_nft_message_add_setelem;
+        sd_nfnl_nft_message_add_setelem_end;
```

Comment:
I think these are not stable yet.

Reply:
Hmm, if https://github.com/systemd/systemd/pull/23774 goes in, I think those won't be used. Otherwise this becomes quite messy to unexport them.